### PR TITLE
add docker support and fix bugs with registration

### PR DESCRIPTION
**Pull request by Denis-Balako**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,8 @@
+POSTGRES_USER=root
+POSTGRES_PASSWORD=Root1234
+POSTGRES_DATABASE=postgresdb
+POSTGRES_LOCAL_PORT=5434
+POSTGRES_DOCKER_PORT=5432
+SPRING_LOCAL_PORT=8088
+SPRING_DOCKER_PORT=8080
+DEBUG_PORT=5005
```
Comment (Dimagaa) [inline .env:8]: Add empty line

**Review by Dimagaa — CHANGES_REQUESTED** [review]
Well done

File: src/main/resources/db/changelog/changes/07-add-users-with-roles-in-user-table.yaml [hunk 2]
```diff
@@ -27,3 +27,12 @@ databaseChangeLog:
               - column:
                   name: role_id
                   valueNumeric: 1
+        - insert:
+            tableName: user_role
+            columns:
+              - column:
+                  name: user_id
+                  valueNumeric: 1
```
Comment (andrii-hoienko) [inline src/main/resources/db/changelog/changes/07-add-users-with-roles-in-user-table.yaml:35]: It is better to use valueComputed

**Review by andrii-hoienko — CHANGES_REQUESTED** [review]
Nice, but some improvements required

**Review by nacenik — APPROVED** [review]
LGTM! 👍

**Pull request merged**